### ISSUE TITLE
none of hotkeys works

### ISSUE THREAD

**Issue by Awannaphasch2016** [KKPMW/vim-sendtowindow#8]
I can't get any keys to work.
is there a way to check that it is being mapped correct?
I did check with :map and it showed that I used the correct mapping.
not sure what is going on.
do you have the same issue?

I did try both default and non-default options.

**Comment by karoliskoncevicius** [comment]
Try the following.

1. Start vim without any configuration:

        vim -u NONE

2. Load the script of this plugin:

        :source yourfolder/vim-sendtowindow/plugin/sendtosplit.vim
    
3. Create a simple map that sends one line to the bottom window:

        nmap Q  <Plug>SendDown_
    
4. Create a new buffer in a separate window below:

        :new

5. write some text in the buffer above

        itext<esc>

6. Place the cursor on the text and try sending it to the buffer below:

        Q

**Comment by Awannaphasch2016** [comment]
is there a quick way to know where is plugin installed? 
I am did some google searching but it seems that I needs to read their document that it would take some time.
you mind point out possible location?
I use vim-plug.
sorry if question is too noob.

**Comment by karoliskoncevicius** [comment]
First just try downloading this file: https://raw.githubusercontent.com/karoliskoncevicius/vim-sendtowindow/master/plugin/sendtosplit.vim and then go over the instructions in the previous comment - use this as a file for the `:source`command. There is all there is to it with "installing" vim plugins.

This will help us establish if this plugin works or not in your case.

**Comment by Awannaphasch2016** [comment]
I got it working for both case: directly source from specific path and use vim-plug to install 
However, I find behavior of sendtowindow in normal quite awkward. 
in my case, when i pressed key for <Plug>SendRight or <Plug>SendUp or <Plug>SendDown or <Plug>SendLeft. I need to press hjkl for command to work.
if i press j/k, line below and above will be sent.
if i press h/l left/right character to the cursor will be sent. 
is this intended behavior?

**Comment by karoliskoncevicius** [comment]
Glad to hear it is working now for you.

As for j/k and h/l behaviour - I hope this doesn't sound rude or something like that - but if you will be using Vim, I would suggest to put some time aside to learn more about Vim first, before installing the plugins.

For your specific case: try these two Vim commands: `dj`and `dl`. See how `dj` deleted a line and `dl` deleted a character? This plugin is consistent with other Vim behaviours, so if you find that weird, you will probably find Vim itself weird too!

In short - once you issue the send command Vim let's you select what you want to send - a line, a word, a paragraph, or something else entirely. If you want shortcuts for that and always send text in lines, just add a shortcut map I showed above:

    nmap Q  <Plug>SendDown_
    
And it will send a line of code down just by pressing `Q`.

**Comment by 3N4N** [comment]
@karoliskoncevicius How do I select a line with vim motion? There is no vim-motion for current line that I know of. If I use j/k, it selects both the current and the line below/above. How do I send just the current line?
